03week/airplane/css/style.css

```diff
@@ -0,0 +1,198 @@
+  body {
+     background-image: url("../img/airline.jpg");
+     background-repeat: no-repeat;
+     background-size: cover;
+  }
```

Comment:
There are a couple ways this could be done differently with this so that it ALWAYS covers the page. With this, as you change the width of the screen sometimes I see whitespace at the bottom.